doc/release-notes.md

```diff
@@ -51,249 +55,251 @@ masternodes. This results in PoSe punishment/banning for outdated masternodes,
-- `createwallet` can create blank, encrypted or watch-only wallets now.
-- Added `private_keys_enabled` in `getwalletinfo`
-- Added `solvable`, `desc`, `ischange` and `hdmasterfingerprint` in `getaddressinfo`
-- Added `desc` in `listunspent`
-
-Dash-specific changes in existing RPCs:
-- Added `quorumIndex` in `quorum getinfo` and `quorum memberof`
-- In rpc `masternodelist` with parameters `full`, `info` and `json` the PoS penalty score of the MN will be returned. For `json` parameter, the field `pospenaltyscore` was added.
+### The new RPCs are:
+- A new `setwalletflag` RPC sets/unsets flags for an existing wallet.
+- The `spork` RPC call will no longer offer both get (labelled as "basic mode") and set (labelled as "advanced mode") functionality. `spork` will now only offer "basic" functionality. "Advanced" functionality is now exposed through the `sporkupdate` RPC call.
+- The `generateblock` RPC call will mine an array of ordered transactions, defined by hex array `transactions` that can contain either transaction IDs or a hex-encoded serialized raw transaction and set the coinbase destination defined by the `address/descriptor` argument.
+- The `mockscheduler` is a debug RPC call that allows forwarding the scheduler by `delta_time`. This RPC call is hidden and will only be functional on mockable chains (i.e. primarily regtest). `delta_time` must be between 0 - 3600.
+- `dumptxoutset` see bitcoin#16899 and the help text. This RPC is used for UTXO snapshot creation which is a part of the assume utxo project
+- `generatetodescriptor` Mine blocks immediately to a specified descriptor. see bitcoin#16943
+
+### The removed RPCs are:
+* The wallet's `generate` RPC method has been removed. `generate` is only used for testing. 
+The RPC call reaches across multiple subsystems (wallet and mining), so is deprecated to simplify the wallet-node
+interface. Projects that are using `generate` for testing purposes should
+transition to using the `generatetoaddress` call, which does not require or use
+the wallet component. Calling `generatetoaddress` with an address returned by
+`getnewaddress` gives the same functionality as the old `generate` method.
+
+### Changes in existing RPCs introduced through bitcoin backports:
+* The `gettxoutsetinfo` RPC call now accept one optional argument (`hash_type`) that defines the algorithm used for calcluating the UTXO set hash, it will default to `hash_serialized_2` unless explicitly specified otherwise. `hash_type` will influence the key that is used to refer to refer to the UTXO set hash.
+* The `generatedescriptor` RPC call has been introduced to allow mining a set number of blocks, defined by argument `num_blocks`, with the coinbase destination set to a descriptor, defined by the `descriptor` argument. The optional `maxtries` argument can be used to limit iterations.
+- `getnodeaddresses` returns peer addresses known to this node. It may be used to connect to nodes over TCP without using the DNS seeds.
```

Comment:
This also was added in 18.0. Was there a change?

Reply:
Not sure, it's very likely we just didn't remove the release-note file